docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -3,56 +3,84 @@
-    - AMD EPYC 7351\@\2.60GHz
-    - 32
-    - 480GB
-  * - v100-sxm2
-    - 24 nodes with 4 GPUs each
-    - Intel Gold 6132\@\2.60Ghz
-    - 28
-    - 187GB
-  * - t4
-    - 2 nodes with 4 GPUs each
-    - Intel Gold 6132\@\2.60Ghz
-    - 28  
-    - 187GB
-  * - a100
-    - 1 node with 4 GPUs
-    - AMD EPYC 7543\@\2.80GHz
-    - 64  
-    - 512GB    
-
-These GPUs are available within two partitions, named ``gpu`` and ``multigpu``. Note that partitions on Discovery are not physical partitions, they  are virtual partitions.
-The differences between the two partitions are the number of GPUs that you can request per job, as well as the time
-limit on each job. Both partitions give you access to all of the above GPU types. See the table below for the differences between the two partitions. See :ref:`partition_names` for more information about all of the partitions on Discovery.
+    - GPU Memory
+    - Tensor Cores
+    - CUDA Cores
+    - Public number of nodes
+    - Private number of nodes
+  * - p100 (`Pascal <https://www.nvidia.com/en-us/data-center/tesla-p100/>`_)
+    - 12GB
+    - N/A 
+    - 3,584
+    - 12 with 3-4 GPUs each
+    - 3 with 4 GPUs each
+  * - v100-pcie (`Volta <https://www.nvidia.com/en-us/data-center/v100/>`_)
+    - 32GB 
+    - 640
+    - 5,120
+    - 4 with 2 GPUs each
+    - 1 with (16GB) 2 GPUs
+  * - v100-sxm2 (Volta)
+    - 32GB
+    - 640
+    - 5,120
+    - 24 with 4 GPUs each
+    - 10 with 4 GPUs each & 16GB GPU memory; 8 with 4 GPUs & 32GB GPU memory
+  * - t4 (`Turing <https://www.nvidia.com/en-us/data-center/tesla-t4/>`_)
+    - 15GB
+    - 320
+    - 2,560
+    - 2 with 3-4 GPUs each
+    - 1 with 4 GPUs
+  * - quadro (`Quadro RTX 8000 <https://www.nvidia.com/en-us/design-visualization/previous-quadro-desktop-gpus/>`_) 
+    - 46GB
+    - 576
+    - 4,608  
+    - 0
+    - 2 with 3 GPUs each
+  * - a30 ( `Ampere <https://www.nvidia.com/en-us/data-center/products/a30-gpu/>`_)
+    - 24GB
+    - 224
+    - 3,804
+    - 0
+    - 1 with 3 GPUs 
+  * - a100 (`Ampere <https://www.nvidia.com/en-us/data-center/a100/>`_)
+    - 41 & 82GB
+    - 432
+    - 6,912  
+    - 3 nodes with 4 GPUs each
+    - 15 nodes with 2-8 GPUs each
+  * - a5000 (`Ampere RTX A5000 <https://www.nvidia.com/en-us/design-visualization/rtx-a5000/>`_)
+    - 24GB
+    - 256
+    - 8,192  
+    - 0
+    - 6 with 8 GPUs each
+  * - a6000 (`Ampere RTX A6000 <https://www.nvidia.com/en-us/design-visualization/rtx-a6000/>`_)
+    - 49GB
+    - 336
+    - 10,752  
+    - 0
+    - 3 with 8 GPUs each
+
+These GPUs are available within two partitions, named ``gpu`` and
+``multigpu``. The differences between the two partitions are the
+number of GPUs that one can request per job and the time limit on each
+job. Both partitions give access to all of the public GPU types
+mentioned above. The table below shows the differences between the two
+partitions. For more information about the partitions on Discovery,
+see :ref:`partition_names`.
```

Comment:
I would incorporate "Both partitions give access to all of the public GPU types
mentioned above." into the first sentence because it reads that both the public and private gpus are available in the gpu and multigpu partition until you get to the line above. 

So maybe something like:

The GPUs listed in the public column are available within two partitions, named "gpu" and "multigpu".

Reply:
Thanks for catching that @joseph-cooke. All done